packages/mapviewer/src/modules/menu/components/header/AdditionalInfoCollapsable.vue

```diff
@@ -0,0 +1,58 @@
+<script setup>
+import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
+import { ref } from 'vue'
+
+import MoreInfo from '@/modules/menu/components/help/MoreInfo.vue'
+import UpdateInfo from '@/modules/menu/components/help/UpdateInfo.vue'
+
+const isOpen = ref(false)
+</script>
+
+<template>
+    <div
+        class="z-menu-1-on-hover"
+        @mouseenter="isOpen = true"
+        @mouseleave="isOpen = false"
+        @click="isOpen = true"
+    >
+        <!-- Menu Button -->
+        <button class="btn m-0 px-1 btn-xs darken-on-hover no-bottom-rounded-corner">
+            <FontAwesomeIcon :icon="isOpen ? 'caret-up' : 'caret-down'" />
+            <span class="ms-1">News</span>
+        </button>
+
+        <!-- Collapsible Menu Items -->
+        <div
+            v-if="isOpen"
+            class="d-flex flex-column-reverse shadow-lg position-absolute"
+            data-cy="news-menu"
+        >
+            <MoreInfo class="darken-on-hover collapsable-menu-element px-1 rounded-0 z-3" />
+            <UpdateInfo class="darken-on-hover collapsable-menu-element px-1 rounded-0 z-3" />
```

Comment:
I'd add a bit more padding around those, maybe a general `p-1` could be enough (and `px-2`), they feel a bit too squeezed by the container as it stands.